src/services/item/plugins/tag/controller.ts

```diff
@@ -37,7 +37,23 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
       await db.transaction(async (manager) => {
         const repositories = buildRepositories(manager);
 
-        await itemToTagService.create(user?.account, repositories, itemId, body);
+        await itemTagService.create(user?.account, repositories, itemId, body);
+      });
+      reply.status(StatusCodes.NO_CONTENT);
+    },
+  );
+
+  fastify.delete(
+    '/:itemId/tags/:tagId',
+    {
+      schema: deleteTagForItem,
+      preHandler: [isAuthenticated, matchOne(validatedMemberAccountRole)],
+    },
+    async ({ user, params: { itemId, tagId } }, reply) => {
+      await db.transaction(async (manager) => {
+        const repositories = buildRepositories(manager);
+
+        await itemTagService.delete(user?.account, repositories, itemId, tagId);
```

Comment:
Here you should probably use the `assertDefined(user?.account)` so you can be sure that the actor is defined. and later use the `Member` type. This makes it more clear to the folowing services that we are dealing with an authentication required service function.